rteeg/stream.py

```diff
@@ -75,9 +75,8 @@ def make_events(data, marker_stream, event_duration=0):
     lower_time_limit = data[-1, 0]
     upper_time_limit = data[-1, -1]
     # Copy markers into a Numpy ndarray.
-    tmp = np.array([row[:] for row in marker_stream.data
-                    if upper_time_limit >= row[-1] >= lower_time_limit],
-                   dtype=np.int32)
```

Comment:
Without sticking to integer values, fine grained times (because of a high sampling rate) don't end up in confusing values after rounding.